jdaviz/configs/default/plugins/subset_tools/tests/test_subset_tools.py

```diff
@@ -384,6 +387,13 @@ def test_get_regions_composite_wcs_linked(imviz_helper, image_2d_wcs):
     assert_allclose(cr.region2.center.ra.deg, sr2.center.ra.deg)
     assert cr.operator == operator.and_
 
+    regs_wcs_with_pixel = st.get_regions(return_sky_region=False,
```

Comment:
i would add some more test coverage here - cover a case where you are WCS linked and want pixel regions therefore you need to set wcs_from_data, a case where you are pixel linked but want sky regions so you need to set wcs_from_data, and then a case where you are wcs linked and want sky regions, but specify wcs_from_data to override the default wcs

Reply:
Please let me know if the tests I added cover these workflows!